services/github/project/api/routes.py

```diff
@@ -200,7 +210,40 @@ def get_items():
                     item['reviewer_points'] = 0
                 item['num_reactions'] = r.get('reactions').get('totalCount') or 0
                 item['title'] = r.get('title')
-                items.append(item)
+                # check if date is between start and end date
+                if start_creation_date != None and end_creation_date != None:
+                    try:
+                        item_date = datetime.datetime.strptime(item['createdAt'].split('T')[0], "%Y-%m-%d")
+                        start_date = datetime.datetime.strptime(start_creation_date, "%Y-%m-%d")
+                        end_date = datetime.datetime.strptime(end_creation_date, "%Y-%m-%d")
+
+                        if check_between_dates(start_date, end_date, item_date):
+                            items.append(item)
+                    except ValueError:
+                        response_object = {
+                            'status': 'fail',
+                            'message': "date format error, format should be YYYY-MM-DD"
+                        }
+                        db.session.rollback()
+                        return jsonify(response_object), 400
+                elif start_updated_date != None and end_updated_date != None:
+                    try:
+                        item_date = datetime.datetime.strptime(item['updatedAt'].split('T')[0], "%Y-%m-%d")
+                        start_date = datetime.datetime.strptime(start_updated_date, "%Y-%m-%d")
+                        end_date = datetime.datetime.strptime(end_updated_date, "%Y-%m-%d")
+
+                        if check_between_dates(start_date, end_date, item_date):
```

Comment:
It'd be neat if this took 2+ args and just made sure they were in ascending order.

Reply:
What's an example of this method needing more than 3 args? I can see the use of having just one date or another (as per your previous comment) but I might be missing the use case of having more than 3 args.

Reply:
Just thought it'd be neat. My original thinking was the args should be in expected ascending order: `start_date, item_date, end_date`. Then I expanded it to 2+ args. I'm fine with just 3 args, but prefer expected ordering.